### PR TITLE
refactor(gnb): allow href as menu endpoint and add navigate event to gnb-sub-menu

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
- Make GNB menu work with the external link
- Add 'navigate' event to gnb-sub-menu

### CODE REVIEW

**Review by piggggggggy — APPROVED** [review]
LGTM!~

File: src/common/modules/navigations/gnb/modules/gnb-menu/GNBSubMenu.vue [hunk 1]
```diff
@@ -3,32 +3,40 @@
...
-                         class="drag-icon"
-                    />
-                    <div ref="labelRef"
-                         class="label"
+            <template #default="{href: toHref, navigate}">
+                <span>
+                    <a class="gnb-sub-contents"
+                       :href="href ? href : toHref"
+                       :target="href ? '_blank' : undefined"
+                       @click.stop="handleClickAnchor(navigate, $event)"
```
Comment (piggggggggy) [inline src/common/modules/navigations/gnb/modules/gnb-menu/GNBSubMenu.vue:15]: 👀

File: src/common/modules/navigations/gnb/modules/gnb-menu/GNBSubMenu.vue [hunk 2]
```diff
@@ -96,9 +109,15 @@ export default defineComponent<Props>({
             } return false;
         };
 
+        const handleClickAnchor = (navigateFn, event: Event) => {
+            if (!props.href) navigateFn(event);
+            emit('navigate');
+        };
```
Comment (piggggggggy) [inline src/common/modules/navigations/gnb/modules/gnb-menu/GNBSubMenu.vue:115]: 👀

**Review by sulmoJ — APPROVED** [review]
LGTM!
`custom` attribute.. It's good!

**Pull request merged**